docs/technical-reference/subtab_architecture/arch-apigateway-01.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
In my head, it depended on the target group and purpose of a section. 
If it's necessary information for the "big overview" and to enable users to use Kyma at all, it should be high up. 
If it's information that only very few people will ever look up, I'd personally rather put it into references (but add links to the matching reference page into any other topic where it might be helpful to extra-curious readers).